.travis/ubuntu-20.04/Dockerfile

```diff
@@ -0,0 +1,31 @@
+FROM ubuntu:focal
```

Comment:
This dockerfile is exactly the same as for debian 10 (and maybe others). Do you think if we can re-use debian 10 dockerfile here just by adding build arg? Like:
```dockerfile
ARG base=debian:buster
FROM ${base}
...
```

And then in CI script:
```bash
docker build --build-arg base="ubuntu:focal" ...
```

Reply:
I can certainly chat that, but do you think the added complexity is worth the reduced duplication?

Reply:
It was just a proposal. Sometimes we need to modify these files and it would be simpler just to edit one file instead. But we can have a separate PR for that if we decide that this is necessary. In my opinion, we can have multiple files as we don't change them very frequently.

Reply:
@mliszcz Okay so I leave it as is for now.